src/components/graph-view.js

```diff
@@ -661,6 +661,11 @@ class GraphView extends Component {
     let src = this.props.getViewNode(edge.source);
     let trg = this.props.getViewNode(edge.target);
 
+    if (!(src && trg)) {
+      console.warn("Unable to get both source and target for ", edge);
+      return "";
+    }
```

Comment:
This is technically sound, but I wonder again in what case this happens. Is it a race condition or can it be created using bad data in the edges?